jujugui/static/gui/src/app/assets/css/_content-panel.scss

```diff
@@ -10,6 +13,57 @@
     background-image: none;
     box-shadow: 0 0 2px rgba(0, 0, 0, 0.3);
 
+    .left-breakout {
+        @extend .type9;
+        z-index: 500;
+        left: auto;
+        right: $inspector-width;
```

Comment:
Do you need z-index, left and right here as they dont seem to change anything once removed?